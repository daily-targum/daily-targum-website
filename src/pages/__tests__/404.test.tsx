import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../404';


describe("<404 />", () => {

  it("renders correctly", () => {
    const tree = renderer.create(
      <NotFound/>
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
  
});