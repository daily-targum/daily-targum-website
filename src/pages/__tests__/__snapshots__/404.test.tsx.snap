// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<404 /> renders correctly 1`] = `
<h1>
  404. Not found.
</h1>
`;
